meta-ivi/recipes-core-ivi/dbus/dbus/build_fix-ubunt1604-qemux86.patch

```diff
@@ -30,15 +9,3 @@
      {
        int *a = dbus_realloc(loader->unix_fds,
                              loader->max_message_unix_fds * sizeof(loader->unix_fds[0]));
---- dbus-1.10.10/dbus/dbus-shell.c.orig	2017-01-26 20:45:49.292201410 +0900
-+++ dbus-1.10.10/dbus/dbus-shell.c	2017-01-26 20:47:39.840204241 +0900
-@@ -419,7 +419,9 @@
-             {
-             case '\n':
-               if (!delimit_token (&current_token, &retval, error))
-+		{
```

Comment:
Skipped this since it serves no purpose.  This is a  style change, that should either just be accepted as it is, or changed upstream, not here.